common.h

```diff
@@ -222,15 +223,14 @@
     LOCAL page_setup_start page_setup_end
     PUSH_EADX
 
-    /* Page directory steup. */
-    /* Set the top 20 address bits. */
+    /* Page Directory setup. */
     mov $page_table, %eax
-    /* Zero out the 4 low flag bits of the second byte (top 20 are address). */
+    /* Clear the low 12 bits of the first Page Directory entry. */
     and $0xF000, %ax
+    /* Set the P, R/W, U/S, and A bits of the first Page Directory entry. */
+    orb $0b00100111, %al
+    /* Setup the first Page Directory entry. */
     mov %eax, page_directory
```

Comment:
Can we condense the above into a single literal mov into eax?

Reply:
Yes. We can merge and $0xF000, %ax and orb $0b00100111, %al into one code. I will refine the code.

Reply:
Do you mean: merge `and $0xF000, %ax` and `orb $0b00100111, %al` into `and $0xF027, %ax`?

Actually, they are different. If `%al` value is `0b***1****`, they will have diff.

Reply:
Ok. I will refine to `mov $0b00100111, %al`.